taskomatic/jobs/cacheman.py

```diff
@@ -81,22 +84,28 @@ def _select_count_affected(account_id, group_ids, all_host_types=False):
                                                                 rule_only = false))) {groups_cond}
             GROUP BY sv.cve_id
             """
-        ).format(host_type_cond=host_type_cond, groups_cond=groups_cond),
+        ).format(edge_patched_cond=edge_patched_cond, edge_unpatched_cond=edge_unpatched_cond, groups_cond=groups_cond),
         [account_id, account_id, account_id, groups_arg],
     )
 
 
 def _select_count_unpatched(account_id, group_ids, all_host_types=False):
-    host_type_cond = _host_type_condition(all_host_types)
+    edge_unpatched_cond = (
+        SQL("COUNT(DISTINCT svp.system_id) FILTER (WHERE sp.host_type = 'edge') AS systems_affected_unpatched_edge,")
+        if all_host_types
+        else SQL("0 AS systems_affected_unpatched_edge,")
+    )
     groups_cond, groups_arg = _inventory_groups_condition(group_ids)
 
     return (
         SQL(
             """
             SELECT vpc.cve_id,
-                   0 AS systems_affected,
```

Comment:
Since `systems_affected` cache is going away, would there be an impact when an `affecting` filter would be used utilizing sum of the two counts (edge + rpmdnf)?

Reply:
Should not be a problem, it is just an easy sum